doc/source/guide/introduction/getting_started.rst

```diff
@@ -2,4 +2,288 @@
 Getting started
 ###############
 
-Welcome to the ODL users guide, this guide is intended to give you a simple introduction to ODL and how to work with it. If you need help on a specific function you should look at its documentation.
\ No newline at end of file
+Welcome to the ODL users guide, this guide is intended to give you a simple introduction to
+ODL and how to work with it. If you need help on a specific function you should look at its
+documentation.
+
+The best way to get started with ODL as a user is generally to find one (or more) examples
+that are relevant to whatver problem you are studying. These are available in the `examples folder on GitHub <https://github.com/odlgroup/odl/tree/master/examples>`_. They are mostly written to be copy-paste friendly
+and show how to use the respective operators, solvers and spaces in a correct manner.
+
+Example: Solving an inverse problem
+===================================
+In what follows, we will give an example of the workflow one might have when solving an inverse problem as
+it is encountered "in real life". The problem we want to solve is
+
+.. math::
+   
+   Af = g
+
+Where :math:`A` is the `convolution <https://en.wikipedia.org/wiki/Convolution>`_ operator
+
+.. math::
+   
+   (Af)(x) = \int f(x) k(x-y) dy
+
+where :math:`k` is the convolution kernel, :math:`f` is the solution and :math:`g` is known data. As is typical in applications, the convolution operator may not be available in ODL (we'll pretend it's not), so we will need to implement it.
```

Comment:
In text documents, I would strongly advocate a "one sentence per line" policy, mainly because that makes it very easy to compare diffs. It also makes it obvious when one repeats oneself a lot at the start of sentences.